### PR TITLE
Add the common content for C#

### PR DESCRIPTION
**Pull request by shavidissa**
Add the common content for C#

### CODE REVIEW

**Comment by shavidissa** [comment]
CC @rkempf42

File: csharp/wavefrontsender.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
+# Set Up an IWavefrontSender Instance
```
Comment (rkempf42) [inline csharp/wavefrontsender.md:1]: Is it really **IWavefrontSender**? just checking

File: csharp/wavefrontsender.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+| `MetricsPort()` | `pushListenerPorts=` |
+| `DistributionPort()` | `histogramDistListenerPorts=` |
+| `TracingPort()` | `traceListenerPorts=` |
+
+## Option 2: Sending Data via Direct Ingestion
+
+When sending data via direct ingestion, you need to create a `WavefrontDirectIngestionClient`, and build it with the Wavefront URL and API token to send data directly to Wavefront.
+
+>**Prerequisite**
+> * Verify that you have the Direct Data Ingestion permission. For details, see [Examine Groups and Permissions](https://docs.wavefront.com/users_account_managing.html#examine-groups-and-permissions).
```
Comment (rkempf42) [inline csharp/wavefrontsender.md:68]: This link will change.

**Review by rkempf42 — APPROVED** [review]
2 very minor comments, otherwise, LGTM

File: csharp/wavefrontsender.md (hunk 1 above)
Comment (shavidissa) [reply csharp/wavefrontsender.md:1]: Yeah .. that's. what the code sample says :)

**Comment by shavidissa** [comment]
cc @vikramraman 
cc @sushantdewan123

**Pull request merged**